packages/core/src/connectivity/scp-cf/jwt.ts

```diff
@@ -315,11 +355,11 @@ export function readPropertyWithWarn(
  * @param token - Encoded JWT as a string.
  * @returns The value of the `jku` property of the JWT header.
  */
-function getVerificationKeyURL(token: string): string {
+function getVerificationKeyUrl(token: string): string {
   const decodedJwt = decodeJwtComplete(token);
-  if (!decodedJwt.header.jku) {
+  if (!decodedJwt.header.jku || !decodedJwt.header.kid) {
```

Comment:
here once more the same test

Reply:
I mean an alternative could be to add another type that has only jku und kid, but it seems a bit like overkill to me. But only a bit, I could also do this. What do you think?

Reply:
yes keep it - will be removed soon when the lib supports cache invalidation.